src/mmw/js/src/account/views.js

```diff
@@ -13,7 +15,64 @@ var Clipboard = require('clipboard'),
     accountTmpl = require('./templates/account.html');
 
 var ProfileView = Marionette.ItemView.extend({
-    template: profileTmpl
+    template: profileTmpl,
+
+    ui: {
+        firstName: '#first_name',
+        lastName: '#last_name',
+        organization: '#organization',
+        userType: '#user_type',
+        country: '#country',
+        postalCode: '#postal_code',
+        saveChanges: '[data-action="save_changes"]'
+    },
+
+    events: {
+        'click @ui.saveChanges': 'saveChanges'
+    },
+
+    modelEvents: {
+        'change': 'render'
+    },
+
+    onRender: function() {
+        var self = this,
+            addOptions = function(model, field, selectedValue) {
+                $.each(window.clientSettings.choices[model][field], function(index, choice) {
+                    var value = choice[0];
+                    var text = choice[1];
+                    var selected = value === selectedValue ? ' selected ' : '';
+                    self.$el.find('#' + field).append('<option ' + selected + ' value="' + value + '">' + text + '</option>');
+                });
+                self.$el.find('#' + field).selectpicker();
+            };
+        addOptions('UserProfile', 'country', self.model.attributes.country);
```

Comment:
Another spot where you might consider using `.get('country')` like we do for the rest of the project.

Reply:
Will do.

Reply:
Addressed in fixup 098ea6f